openproblems/tasks/spatial_decomposition/__init__.py

```diff
@@ -0,0 +1,11 @@
+from ... import utils
+from . import api
+from . import datasets
+from . import methods
+from . import metrics
+
+_task_name = "Label Projection"
```

Comment:
We want to change the task name.

Reply:
Well spotted, this is now taken care of!